src/app/hooks/useWalletReducer.ts

```diff
@@ -254,7 +254,7 @@ const useWalletReducer = () => {
   const switchAccount = async (account: Account) => {
     // we clear the query cache to prevent data from the other account potentially being displayed
     await queryClient.cancelQueries();
-    await queryClient.clear();
+    queryClient.clear();
```

Comment:
this was a type warning. queryClient.clear() is not async

Reply:
👍 I think this was my bad 😅